ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/services/reset/ResetServiceITBase.scala

```diff
@@ -59,6 +60,7 @@ import org.scalatest.{AsyncWordSpec, Matchers}
 import scala.concurrent.Future
 import scala.concurrent.duration.{DurationInt, DurationLong, FiniteDuration}
 
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
```

Comment:
Could you add a comment explaining why this is necessary?

Reply:
Rewrote the assertion to not trigger the `Any` wart: ff0d753